### PR TITLE
CU-86drvtfez - NEON3 - Settings Screen - General design fixes

### PR DESCRIPTION
**Pull request by Thacryba**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by melanke** [comment]
Task linked: [CU-86drvtfez NEON3 - Settings Screen - General design fixes ](https://app.clickup.com/t/86drvtfez)

File: src/renderer/src/routes/pages/Settings/SettingsNetwork/index.tsx [hunk 1]
```diff
@@ -31,41 +31,38 @@ export const SettingsNetwork = () => {
         <h1 className="text-white text-sm">{t('title')}</h1>
 
         <Button
-          leftIcon={<MdRestartAlt className="text-neon" />}
+          leftIcon={<MdRestartAlt className="text-neon w-6 h-6" />}
           label={commonGeneral('reset')}
-          className="w-fit"
-          variant="text"
+          className="w-fit mr-5"
+          variant="text-slim"
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Settings/SettingsNetwork/index.tsx]: You can let the variant as "text" and change the `<Clickable />` styles. The `<Base />` has a `h-12`, you can change it to `h-10`

File: src/renderer/src/routes/pages/Settings/SettingsNetwork/index.tsx [hunk 2]
```diff
@@ -31,41 +31,38 @@ export const SettingsNetwork = () => {
...
           colorSchema="gray"
           onClick={handleChangeNetwork.bind(null, 'mainnet')}
         />
       </header>
 
-      <div className="mt-8 mb-5 text-sm">{t('youAreConnectedNeoAndEth')}</div>
+      <div className="mt-[1.829rem] mb-7 text-xs">{t('youAreConnectedNeoAndEth')}</div>
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Settings/SettingsNetwork/index.tsx]: Use `my-7 `instead `mt-[1.829rem] mb-7`

File: src/renderer/src/routes/pages/Settings/index.tsx [hunk 3]
```diff
@@ -26,17 +26,17 @@ export const SettingsPage = () => {
   return (
     <MainLayout heading={t('title')}>
       <section className="bg-gray-800 w-full h-full flex rounded">
-        <div className="min-w-[17.5rem] max-w-[17.5rem] px-5 border-r flex flex-col items-center">
+        <div className="min-w-[17.5rem] max-w-[17.5rem] px-5 border-r border-gray-300/15 flex flex-col items-center">
           <Tabs.Root defaultValue={ESettingsOptions.PERSONALISATION} className="w-full">
-            <Tabs.List className="w-full mt-2.5 mb-7">
+            <Tabs.List className="w-full mt-[0.9rem] mb-7">
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Settings/index.tsx]: To align the two headers, leave it as it was and apply an h-[3.25rem] to the SettingsNetwork header. Try to apply this solutions for  the contacts page headers as well, so we will have just one solution for it

**Pull request merged**